CHANGELOG.md

```diff
@@ -12,10 +12,10 @@ Possible log types:
 - `[security]` to invite users to upgrade in case of vulnerabilities.
 
 
-### UNRELEASED
+### v0.3.0 UNRELEASED
```

Comment:
@dbrgn I changed it to 0.3 here, since it is clear now that we will release 0.3.